onnx/defs/tensor/defs.cc

```diff
@@ -1797,62 +1797,115 @@ ONNX_OPERATOR_SET_SCHEMA(
 
 static const char* Unique_ver11_doc = R"DOC(
 Finds all the unique values (deduped list) present in the given input tensor. This operator returns 3 outputs. 
-The first output tensor 'uniques' contains all of the unique elements of the input, sorted in the same order that they occur in the input.
+The first output tensor 'uniques' contains all of the unique elements of the input, 
+optionally sorted in ascending or the same order that they occur in the input. 
```

Comment:
Please add description of "inverse_indices" output also.

Reply:
updated comment to fix this.